source/resource/blocks.json

```diff
@@ -87,8 +87,111 @@
           }
         ]
       },
-      "sound": { "type": "string", "title": "Sound", "description": "The sound definition of this block." },
-      "textures": { "$ref": "#/definitions/texture", "title": "Sound", "description": "Textures." }
+      "sound": { 
+        "type": "string", 
+        "title": "Sound", 
+        "description": "The sound definition of this block.", 
+        "enum": [
+          "amethyst_block",
+          "amethyst_cluster",
+          "ancient_debris",
+          "anvil",
+          "azalea",
+          "azalea_leaves",
+          "bamboo",
+          "bamboo_sapling",
+          "bamboo_wood",
+          "bamboo_wood_hanging_sign",
+          "basalt",
+          "big_dripleaf",
+          "bone_block",
+          "calcite",
+          "candle",
+          "cave_vines",
+          "chain",
+          "cherry_leaves",
+          "cherry_wood",
+          "cherry_wood_hanging_sign",
+          "chiseled_bookshelf",
+          "cloth",
+          "comparator",
+          "copper",
+          "copper_bulb",
+          "copper_grate",
+          "coral",
+          "decorated_pot",
+          "deepslate",
+          "deepslate_bricks",
+          "dirt_with_roots",
+          "dripstone_block",
+          "frog_spawn",
+          "froglight",
+          "fungus",
+          "glass",
+          "grass",
+          "gravel",
+          "hanging_roots",
+          "hanging_sign",
+          "heavy_core",
+          "honey_block",
+          "itemframe",
+          "ladder",
+          "lantern",
+          "large_amethyst_bud",
+          "lever",
+          "lodestone",
+          "mangrove_roots",
+          "medium_amethyst_bud",
+          "metal",
+          "moss_block",
+          "moss_carpet",
+          "mud",
+          "mud_bricks",
+          "muddy_mangrove_roots",
+          "nether_brick",
+          "nether_gold_ore",
+          "nether_sprouts",
+          "nether_wart",
+          "nether_wood",
+          "nether_wood_hanging_sign",
+          "netherite",
+          "netherrack",
+          "nylium",
+          "packed_mud",
+          "pink_petals",
+          "pointed_dripstone",
+          "polished_tuff",
+          "powder_snow",
+          "roots",
+          "sand",
+          "scaffolding",
+          "sculk",
+          "sculk_catalyst",
+          "sculk_sensor",
+          "sculk_shrieker",
+          "sculk_vein",
+          "shroomlight",
+          "slime",
+          "small_amethyst_bud",
+          "snow",
+          "soul_sand",
+          "soul_soil",
+          "spore_blossom",
+          "stem",
+          "stone",
+          "suspicious_gravel",
+          "suspicious_sand",
+          "sweet_berry_bush",
+          "trial_spawner",
+          "tuff",
+          "tuff_bricks",
+          "vault",
+          "vines",
+          "web",
+          "wood"
+        ]

```

Comment:
Sorry cant have this as an enum as its able to have custom enums, this should be done through the json path completion in the vscode extension ☺️

Reply:
I thaught so 🤔

Reply:
It's limited to vanilla sounds unfortunately